### ISSUE TITLE
no zulip notifications

### ISSUE THREAD

**Issue by ehuss** [rust-lang/triagebot#1468]
I noticed that Zulip notifications have not been working this morning.

Config: https://github.com/rust-lang/rust/blob/84b018341284798fa47f8171f4eb04f18f45cf23/triagebot.toml#L115-L121
Topic: https://rust-lang.zulipchat.com/#narrow/stream/268952-edition-2021/topic/Edition.20Bugs
Example issues: https://github.com/rust-lang/rust/issues/88431 https://github.com/rust-lang/rust/issues/88429

**Comment by Mark-Simulacrum** [comment]
Caused by c0e3c8d8873fa22c389033a47abd78fdc56614b0 which made IssueRepository fail to deserialize from webhooks:

request failed: WebhookError(Issue failed to deserialize

Caused by:
    0: at Path { segments: [Map { key: "repository" }, Map { key: "owner" }] }
    1: invalid type: map, expected a string at line 1 column 6536)

cc @jackh726 

Going to work on a fix now.

**Comment by jackh726** [comment]
Oh. Hmm

**Comment by jackh726** [comment]
@Mark-Simulacrum I can submit a fix for this right now if you don't have one

**Comment by Mark-Simulacrum** [comment]
@jackh726 please take a look at my pr, but I think it should fix this. Happy to consider alternatives though, not sure if my work breaks the graphql anywhere.

**Comment by jackh726** [comment]
Looks good to me. Graphql types are generated directly from the schema so can't really break.

**Comment by ehuss** [comment]
Thanks so much for the fast response!

However, it doesn't seem to be working again.  There was one notification a little while after this was fixed, but I've filed two more issues since then which have not triggered anything: https://github.com/rust-lang/rust/issues/88442 and https://github.com/rust-lang/rust/issues/88444.  What's strange is that https://github.com/rust-lang/rust/issues/88440 made a notification, and that was filed only about 15 minutes before 88442.
